development/hub/run-debug.sh

```diff
@@ -13,5 +23,7 @@ echo "xdebug.remote_host=$XDEBUG_REMOTE_HOST" >> $INI_FILE
 mkdir -p /data/vendor/simplesamlphp/simplesamlphp/modules/sildisco
 touch /data/vendor/simplesamlphp/simplesamlphp/modules/sildisco/enable
 
+php -r "require 'vendor/autoload.php'; echo(PHP_EOL . 'Aws\Sdk::VERSION = ' . Aws\Sdk::VERSION . ' <<<' . PHP_EOL);"
```

Comment:
Out of curiosity, why the `' <<<'` string? At first, I thought it was some bit of bash command sending or reading content to/from somewhere.

Reply:
That was just to make it clear where the end of the version was.  I guess at one point I was worried about it coming back as an empty string.

Reply:
I would, but just for the sake of simplifying things. Either way seems fine.

Reply:
Removed